ddgr

```diff
@@ -1503,9 +1503,12 @@ class DdgCmd(object):
                     try:
                         # try copying the url to clipboard using native utilities
                         if sys.platform.startswith(('linux', 'freebsd', 'openbsd')):
-                            if shutil.which('xsel') is None:
+                            if shutil.which('xsel') != None:
```

Comment:
Should be `is not None`, lint complains.

Same for line 1508.

Reply:
Totally forgot about linting. Just pushed a commit.